goodtablesio/tests/plugins/github/test_tasks.py

```diff
@@ -1,17 +1,24 @@
 from unittest.mock import patch
 from goodtablesio.plugins.github import tasks
 
+# Constants
+
+OWNER = 'frictionlessdata'
+REPO = 'goodtables.io-example'
+SHA = 'd5be243487d9882d7f762e7fa04b36b900164a59'
+
 
 # Tests
 
 def test_get_job_base():
-    actual = tasks._get_job_base('frictionlessdata', 'goodtables.io-example')
-    expect = 'https://raw.githubusercontent.com/frictionlessdata/goodtables.io-example/master'  # noqa
+    actual = tasks._get_job_base(OWNER, REPO, SHA)
+    expect = 'https://raw.githubusercontent.com/frictionlessdata/'
+    expect += 'goodtables.io-example/d5be243487d9882d7f762e7fa04b36b900164a59'
     assert actual == expect
 
 
 def test_get_job_files():
-    actual = tasks._get_job_files('frictionlessdata', 'goodtables.io-example', 'd5be243487d9882d7f762e7fa04b36b900164a59')  # noqa
+    actual = tasks._get_job_files(OWNER, REPO, SHA)
```

Comment:
It's not an error in the code, it's an error in the test: `'LICENSE.md'` was not in the expected files. We are testing a static list against something that can change externally (ie the repo where we can add or delete files). I remain convinced that mocking is better here, but yes, no need to change it now (unless it breaks again ;) ). BTW if I can avoid it I prefer not use `httpretty` and use plain mocking as it has caused me many problems in the past.